concepts/Common/Security Considerations/20 HTML Encoding/20 Encoding Across Components/encodeHtml.md

```diff
@@ -0,0 +1,24 @@
+**encodeHtml** is a Boolean option that you can set for a column in the [DataGrid](/Documentation/ApiReference/UI_Widgets/dxDataGrid/) and [TreeList](/Documentation/ApiReference/UI_Widgets/dxTreeList/) components. Its default value is **true**, which means that column values are encoded. If you set it to **false**, the encoding is disabled, and malicious code can be executed. We recommend keeping this option set to **true**.
```

Comment:
```suggestion
**encodeHtml** is a Boolean option that you can set for a column in the [DataGrid](/Documentation/ApiReference/UI_Widgets/dxDataGrid/) and [TreeList](/Documentation/ApiReference/UI_Widgets/dxTreeList/) components. Its default value is **true**, which means that column values are encoded. If you set it to **false**, the encoding is disabled, and malicious code can be executed. We recommend that you set this option to **true**.
```

Reply:
This option is set to `true` by default.

Reply:
```suggestion
**encodeHtml** is a Boolean option that you can set for a column in the [DataGrid](/Documentation/ApiReference/UI_Widgets/dxDataGrid/) and [TreeList](/Documentation/ApiReference/UI_Widgets/dxTreeList/) components. Its default value is **true**, which means that column values are encoded. If you set it to **false**, the encoding is disabled, and malicious code can be executed. We recommend that you keep this option set to **true**.
```